components/event-feed-service/pkg/server/server.go

```diff
@@ -164,51 +173,41 @@ func FromInternalFormatToList(entries []*util.FeedEntry) ([]*event_feed.FeedEntr
 func validateFeedTimelineRequest(req *event_feed.FeedTimelineRequest) error {
 	// Validate Timezone
 	if req.Timezone == "" {
-		return errors.GrpcError(codes.InvalidArgument, "A timezone must be provided")
+		return e.GrpcError(codes.InvalidArgument, "A timezone must be provided")
```

Comment:
Whatever it stands for, this should be `GRPCError`, I think. Fine to leave as-is, though. For consistency, don't we want `s/A timezone/a timezone/`?